AK/FixedArray.h

```diff
@@ -15,40 +17,60 @@ namespace AK {
 template<typename T>
 class FixedArray {
 public:
-    FixedArray() = default;
-    explicit FixedArray(size_t size)
-        : m_size(size)
+    explicit FixedArray() = default;
```

Comment:
```suggestion
    FixedArray() = default;
```

No need for `explicit` on an argument-less ctor (& the diff shrinks again :^))